components/items/ItemsGrid/useItemsGrid.ts

```diff
@@ -63,7 +69,13 @@ export function useFetchSearch({
           priceMax: Number(route.query.max),
         }
 
-    const queryPath = getQueryPath(client.value)
+    const queryPathBase = getQueryPath(client.value)
+    const usingTokenEntities = computed(
+      () => isExploreItems.value && queryPathBase === 'ahk'
```

Comment:
Why is it only used on the explore page?

Reply:
See the isuue.
Exec asked for only explore page
If it were to be in collection page as well there would be no way to see individual items

But mostly coz @exezbcz said so

Reply:
Correct, no listings tab - you need to see the individual items somewhere